tests/test_inspect.py

```diff
@@ -87,7 +87,7 @@ def test_get_dataset_info(path, expected_configs, expected_splits_in_first_confi
     assert expected_config in infos
     info = infos[expected_config]
     assert info.config_name == expected_config
-    assert list(info.splits.keys()) == expected_splits_in_first_config
+    assert sorted(info.splits.keys()) == sorted(expected_splits_in_first_config)
```

Comment:
I'm not sure we want to avoid testing the order: as already discussed by you @polinaeterna and @severo, splits are not alphabetically sorted. 

Therefore, it makes sense to test that the order returned by `get_dataset_infos` is the expected one.

Anyway, if finally we decide to sort them, we should do it also in `test_get_dataset_config_info`, which was also failing besides `test_get_dataset_info` and `test_get_dataset_split_names`.

Reply:
yes, agree with you! reverted sorting in tests [a063c6f](https://github.com/huggingface/datasets/pull/5201/commits/a063c6f4b407d5e3bafe789692ca22208da320d1)